src/riscv-legacy-integration.adoc

```diff
@@ -189,8 +189,9 @@ When the XLEN-bit alias is used by <<CSRRW>>:
 * Only XLEN bits from the *x* source are written to the capability address
 field.
     ** The tag and metadata are updated as specified in <<extended_CSR_writing>>.
-* Only XLEN bits are read from the capability address field, which is zero
-extended to the destination *x* register.
+* Only XLEN bits are read from the capability address field, which are extended
+to XLENMAX bits according to cite:[riscv-unpriv-spec] and are then written
```

Comment:
@tariqkurd-repo: Thanks for the change. However, I think the relevant information is actually in the privileged spec. It would also be helpful to point out the Section 3.1.6.2 as @sorear said.

Reply:
I think it's unwise to point out an actual section number as that is subject to change over time.